shapely/predicates.py

```diff
@@ -747,7 +747,7 @@ def equals(a, b, **kwargs):
 def intersects(a, b, **kwargs):
     """Returns True if A and B share any portion of space.
 
-    Intersects implies that overlaps, touches and within are True.
+    Note that touching geometries also intersect.
```

Comment:
@theroggy I like your change, it's more clear. For the sake of completeness, can you restore the mention of overlaps and within?